packages/client/hmi-client/src/App.vue

```diff
@@ -22,14 +22,59 @@
 			/>
 		</router-view>
 	</main>
-	<footer>
+	<footer class="footer">
 		<img src="@assets/svg/uncharted-logo-dark.svg" alt="logo" class="ml-2" />
+		<div class="footer-button-group">
+			<Button label="about" class="footer-button" text @click="visible = true" />
+			<a href="https://terarium.canny.io/report-an-issue" class="no-text-decoration">
+				<Button label="report an issue" class="footer-button" text />{{ null }}
+			</a>
+			<a href="https://terarium.canny.io/request-a-feature" class="no-text-decoration">
+				<Button label="request a feature" class="footer-button" text />{{ null }}
+			</a>
+		</div>
 	</footer>
+
+	<!-- About modal -->
+	<Dialog v-model:visible="visible" modal header="About" :style="{ width: '40vw' }">
+		<div class="about-modal-content">
+			<div class="about-section">
+				<img src="@/assets/svg/terarium-logo.svg" alt="Terarium logo" class="about-terarium-logo" />
+				<p>Version 0.0.1</p>
+			</div>
+			<div class="about-section">
+				<p class="constrain-width">
+					Terarium is a web-based solution featuring human-in-the-loop interactive data
+					visualization for exploring, interpreting, searching, analyzing, viewing and reporting
+					data via multi-dimensional visual representations. Terarium is also designed to support
+					collaboration and interaction relating to data visualization, data mining and analysis.
+				</p>
+			</div>
+			<div class="about-section">
+				<img
+					src="@/assets/svg/uncharted-logo-official.svg"
+					alt="Uncharted Software logo"
+					class="about-uncharted-logo"
+				/>
+				<p class="constrain-width">
+					Uncharted Software provides design, development and consulting services related to data
+					visualization and analysis software.
+				</p>
+				<br />
+				<p>&copy; Copyright Uncharted Software {{ new Date().getFullYear() }}</p>
+			</div>
+		</div>
+		<template #footer>
+			<Button label="Close" @click="visible = false" />
+		</template>
+	</Dialog>
```

Comment:
We have a `<Modal>` component `/src/components/widgets/Modal.vue`

Reply:
Switched to the Modal component. Thanks.